snuba/subscriptions/entity_subscription.py

```diff
@@ -14,6 +14,8 @@
     SubscriptionAllowedClausesValidator,
 )
 
+initialize_entity_factory()
```

Comment:
Hack?: `entity_subscription.py` stuff requires all entities to be loaded so I'm not sure where else to initialize. Tests fail if entity factory isn't initialized otherwise.

Reply:
I'd like to see this file mostly go away -- these just seem like properties that should live on Entity to me

Reply:
Yes this is a hack. The thing that it supports is also a hack. Put the two hacks near each other and encase it in a comment block that explains the hack and file a ticket to remove it

https://github.com/getsentry/snuba/blob/53fadadab46b75aa0bf456c5e1d256eeb962fca3/snuba/subscriptions/entity_subscription.py#L146-L157